git_machete/tests/functional/test_machete.py

```diff
@@ -36,18 +36,48 @@ def __init__(self) -> None:
         self.opt_yes: bool = True
 
 
-class GitAPIState:
-    pulls: List[Dict[str, Any]] = [{'head': {'ref': 'bugfix/remove'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'develop'}, 'number': '15', 'html_url': 'www.github.com'}]
-    user: Dict[str, str] = {'login': 'github_user', 'type': 'User', 'company': 'VirtusLab'}
-    issues: List[Dict[str, Any]] = []
-    remote_branches: List[str] = []
+class MockGitAPIState:
```

Comment:
Is this mock common for all git based services like gitlab and github or is it only for github? If it's only for github it could be a good idea to call it `MockGithubAPIState`.

Reply:
Yeah, things like issue and pull request are unknown to git as such, these are Github-specific terms... so it should definitely  be `MockGithubAPIState/Request`